<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>电刺激仪闭环控制</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="currentPage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QComboBox" name="comBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="rectPulse">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>矩形波</string>
      </property>
     </widget>
     <widget class="QPushButton" name="connect">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
     <widget class="QPushButton" name="singlePulse">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>单脉冲</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="isCollection">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>220</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>采集角度数据</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>90</y>
        <width>161</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>幅值 [mA]</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="sCurrent">
         <property name="maximum">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>脉宽 [us]</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="sWidth">
         <property name="minimum">
          <number>20</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>持续时间 [s]</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="stimulationTime">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="channels.qrc">:/images/pic/1.bmp</pixmap>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/pic/2.bmp&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/pic/3.bmp&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/pic/4.bmp&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/pic/5.bmp&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/pic/6.bmp&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/pic/7.bmp&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/pic/8.bmp&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioButton_1">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string/>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QRadioButton" name="radioButton_3">
         <property name="text">
          <string/>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QRadioButton" name="radioButton_4">
         <property name="text">
          <string/>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QRadioButton" name="radioButton_5">
         <property name="text">
          <string/>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QRadioButton" name="radioButton_6">
         <property name="text">
          <string/>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QRadioButton" name="radioButton_7">
         <property name="text">
          <string/>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QRadioButton" name="radioButton_8">
         <property name="text">
          <string/>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="sineWave">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>210</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>正弦波</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sine_1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>290</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>100</number>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="value">
       <number>300</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="sine_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>290</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sine_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>290</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <double>6.280000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>270</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>偏差</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>270</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>幅值</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>270</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>周期</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>270</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>正弦函数
参数设定</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="ControlStart">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>280</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="customPlot" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>481</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>290</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="sCurrent_2">
      <property name="geometry">
       <rect>
        <x>183</x>
        <y>20</y>
        <width>76</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>77</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>幅值 [mA]</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>期望轨迹</string>
      </property>
     </widget>
     <widget class="QComboBox" name="desiredTrajectory">
      <property name="geometry">
       <rect>
        <x>365</x>
        <y>18</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QCustomPlot" name="anglePlot" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>251</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="widthPlot" native="true">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>60</y>
        <width>251</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>角度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>30</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>脉宽</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>577</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>电刺激页面</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>数据显示页面</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>闭环控制页面</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="channels.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
